scripts/install_zds.sh

```diff
@@ -262,6 +262,18 @@ if  ! $(_in "-node" $@) && ( $(_in "+node" $@) || $(_in "+base" $@) || $(_in "+f
             echo $ACTIVATE_NVM >> $ZDS_ENV/bin/activate
             echo $ACTIVATE_NVM >> $ZDS_ENV/bin/activate.csh
             echo $ACTIVATE_NVM >> $ZDS_ENV/bin/activate.fish
+
+            # bash compatibility for nvm with fish
+            # see https://eshlox.net/2019/01/27/how-to-use-nvm-with-fish-shell
+            if [[ $SHELL =~ "fish" ]]; then
```

Comment:
Modifie tout le temps le .fish non ?

Reply:
J'ai pas compris ta question, tu peux la préciser ? Je ne sais pas à quoi fait référence le .fish.
La condition utilise le variable d’environnement `$SHELL` qui contient le chemin du shell pour l'utilisateur courant et y applique une regex. Dans l'état, elle sera toujours vraie mais elle est conditionnée par les conditions dans les blocs au-dessus d'elle. Donc, elle est sensée être exécutée 1 fois lors de la 1ère installation.

Reply:
echo $ACTIVATE_NVM >> $ZDS_ENV/bin/activate.fish

Est toujours modifié donc ton code doit toujours modifié ce fichier pour qu'il soit fonctionnel.

Reply:
As-tu regardé le lien que j'ai mis dans la PR ? Lien que tu m'avais trouvé d'ailleurs.

Reply:
Je ne dois pas comprendre ce que fais la condition et pourquoi tu l'as mise

Reply:
Je pense aussi... Ça ne répond pas à ma question si tu as lu ou pas.

J'ai mis un commentaire dans mon code `bash compatibility for nvm with fish` et c'est ce que j'explique au début de ma PR.

Reply:
Oui justement, dans l'article il ne parle pas d'installer fisher

Reply:
Ok, on fait comment alors ?

Reply:
Ce que je ne comprends pas c'est pourquoi tu n'appliques pas ton changement à chaque fois même si pendant l'installation j'utilise bash, si jamais je change d'avis un jour j'aurais le fichier fish deja disponible